vip_spec.xsd

```diff
@@ -5,6 +5,120 @@ added ocdid to district
 reference elements for polling location
 -->
 <xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" version="5.0.1" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <!-- Start enumeration types. -->
+  <xs:simpleType name="BallotMeasureType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="ballot-measure" />
+      <xs:enumeration value="initiative" />
+      <xs:enumeration value="referendum" />
+      <xs:enumeration value="retention" />
+      <xs:enumeration value="other" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="CandidatePostElectionStatus">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="advanced-to-runoff" />
+      <xs:enumeration value="projected-winner" />
+      <xs:enumeration value="winner" />
+      <xs:enumeration value="withdrawn" />
+      <xs:enumeration value="write-in" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="CandidatePreElectionStatus">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="filed" />
+      <xs:enumeration value="qualified" />
+      <xs:enumeration value="withdrawn" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
+      <xs:enumeration value="county" />
+      <xs:enumeration value="judicial" />
+      <xs:enumeration value="local" />
+      <xs:enumeration value="locality" />
+      <xs:enumeration value="municipality" />
+      <xs:enumeration value="national" />
+      <xs:enumeration value="school" />
+      <xs:enumeration value="special" />
+      <xs:enumeration value="state" />
+      <xs:enumeration value="statehouse" />
+      <xs:enumeration value="statesenate" />
+      <xs:enumeration value="township" />
+      <xs:enumeration value="utility" />
+      <xs:enumeration value="ward" />
+      <xs:enumeration value="water" />
+      <xs:enumeration value="other" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="IdentifierType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="ocd-id" />
+      <xs:enumeration value="fips" />
+      <xs:enumeration value="national-level" />
+      <xs:enumeration value="state-level" />
+      <xs:enumeration value="local-level" />
+      <xs:enumeration value="other" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="OebEnum">
```

Comment:
Sorry if repeating, but yes, this can happen subsequently.  In this PR, I am restricting only to moving the chunks in this PR as is -- and leaving changes within type definitions to subsequent PR's.  This will make it easier to see and review changes within type definitions, as opposed to having moves combined with changes in the same PR.